Wox.Test/FuzzyMatcherTest.cs

```diff
@@ -48,6 +49,8 @@ public void MatchTest()
                 "aac"
             };
 
+            Alphabet.Initialize(false);
+            StringMatcher.UserSettingSearchPrecision = StringMatcher.SearchPrecisionScore.Low.ToString();
```

Comment:
Is this required? If i remember correctly the default for precisionscore level is regular, otherwise what's the reasoning for setting it to Low